app/src/main/java/com/example/coinage/fragments/AddTransactionFragment.java

```diff
@@ -111,5 +122,71 @@ public void done(com.parse.ParseException e) {
                 Log.i(TAG, "purchase added to database");
             }
         });
+
+        // update spending categories
+        saveSpending(currentUser, category, amount);
+    }
+
+    private void saveSpending(ParseUser currentUser, String category, BigDecimal amount) {
+        ParseQuery<Spending> spendingQuery = ParseQuery.getQuery(Spending.class);
+        spendingQuery.whereEqualTo(Spending.KEY_CATEGORY, category);
+        spendingQuery.findInBackground(new FindCallback<Spending>() {
+            @Override
+            public void done(List<Spending> spendings, com.parse.ParseException e) {
```

Comment:
`com.parse.ParseException` import this at the top instead

Reply:
how do I do this? if I move the com.parse.ParseException out of the parentheses I get an error saying I need to create a type parameter